mzLib/MzLibUtil/PpmToleranceWithNotch.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MzLibUtil
+{
+    public class PpmToleranceWithNotch : Tolerance
+    {
+        private readonly double[] AcceptableSortedMassShifts;
+        public const double NotchStep = 1.00335483810;
+        public PpmTolerance PpmTolerance { get; init; }
+
+        public PpmToleranceWithNotch(double value, int protonNotches)
```

Comment:
This means we will go +- protonNotches always. Is this the intended behavior? Do you want to define positive and negative notch steps?